pyproject.toml

```diff
@@ -43,7 +43,40 @@ Documentation = "https://more-itertools.readthedocs.io/en/stable/"
 [tool.flit.module]
 name = "more_itertools"
 
-[tool.black]
-line-length = 79
-target-version = ["py39"]
-skip-string-normalization = true
+[project.optional-dependencies]
+lint = [
+    "ruff >= 0.8",
+]
+
+[tool.ruff]
+indent-width = 4
+line-length = 80
+target-version = "py39"
+
+[tool.ruff.lint]
+# https://docs.astral.sh/ruff/rules/
+select = [
+    "B",  # flake8-bugbear
+    "E",  # pycodestyle errors
+    "ERA",  # eradicate
+    "F",  # pyflakes
+    "I",  # isort
+    "RET",  # flake8-return
+    "SLF",  # flake8-self
+    "TID",  # flake8-tidy-imports
+    "UP",  # pyupgrade
+]
+ignore = ["B904", "B028", "E741"]
```

Comment:
I understood about this set of rules: the code was linted with the `ruff` default configuration (just guessing) and there were enough errors to make them ignored (for now).